From 3ab119528a7079111abe5d916d3f71f8d08355f1 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Fri, 21 May 2021 20:09:47 -0700
Subject: [PATCH 105/196] Fix two bitcoin-tx transactions amount denominations
 (#180)

Two bitcoin-tx json test files were not updated in the previous
redenominate PR due to a rebase mistake. This commit fixes the newly
named files, and deletes the old ones.
---
 ...reatesignbip341v2-single-anyonecanpay.json | 34 -------------------
 .../data/txcreatesignbip341v2-single.json     | 34 -------------------
 ...createsignlotusv2-single-anyonecanpay.json |  2 +-
 .../util/data/txcreatesignlotusv2-single.json |  2 +-
 4 files changed, 2 insertions(+), 70 deletions(-)
 delete mode 100644 test/util/data/txcreatesignbip341v2-single-anyonecanpay.json
 delete mode 100644 test/util/data/txcreatesignbip341v2-single.json

diff --git a/test/util/data/txcreatesignbip341v2-single-anyonecanpay.json b/test/util/data/txcreatesignbip341v2-single-anyonecanpay.json
deleted file mode 100644
index 032c5261d..000000000
--- a/test/util/data/txcreatesignbip341v2-single-anyonecanpay.json
+++ /dev/null
@@ -1,34 +0,0 @@
-{
-    "txid": "703b165ed20cffab0a5df5ad38b77a93b6e1e250e7a1d6d18162b546ab67f452",
-    "hash": "992e7ac4ebba029956f6df08817de49a6ccd7b296d6f68bf8b7a6846fab82eaf",
-    "version": 2,
-    "size": 191,
-    "locktime": 0,
-    "vin": [
-        {
-            "txid": "4d49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc59485",
-            "vout": 0,
-            "scriptSig": {
-                "asm": "3044022077f44e39e3a3ae076341efae954f84a10211576fffc0106e20d979bcb8aeb665022026f765d100c4b3f0e1f3482a676fd60f725d81d5afd2172adfa23281bbdff97d[SINGLE|BIP341|ANYONECANPAY] 02c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5",
-                "hex": "473044022077f44e39e3a3ae076341efae954f84a10211576fffc0106e20d979bcb8aeb665022026f765d100c4b3f0e1f3482a676fd60f725d81d5afd2172adfa23281bbdff97de32102c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5"
-            },
-            "sequence": 4294967295
-        }
-    ],
-    "vout": [
-        {
-            "value": 0.100000,
-            "n": 0,
-            "scriptPubKey": {
-                "asm": "OP_DUP OP_HASH160 06afd46bcdfd22ef94ac122aa11f241244a37ecc OP_EQUALVERIFY OP_CHECKSIG",
-                "hex": "76a91406afd46bcdfd22ef94ac122aa11f241244a37ecc88ac",
-                "reqSigs": 1,
-                "type": "pubkeyhash",
-                "addresses": [
-                    "1cMh228HTCiwS8ZsaakH8A8wze1JR5ZsP"
-                ]
-            }
-        }
-    ],
-    "hex": "02000000018594c5bdcaec8f06b78b596f31cd292a294fd031e24eec716f43dac91ea7494d000000006a473044022077f44e39e3a3ae076341efae954f84a10211576fffc0106e20d979bcb8aeb665022026f765d100c4b3f0e1f3482a676fd60f725d81d5afd2172adfa23281bbdff97de32102c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5ffffffff01a0860100000000001976a91406afd46bcdfd22ef94ac122aa11f241244a37ecc88ac00000000"
-}
diff --git a/test/util/data/txcreatesignbip341v2-single.json b/test/util/data/txcreatesignbip341v2-single.json
deleted file mode 100644
index abfc1a6c2..000000000
--- a/test/util/data/txcreatesignbip341v2-single.json
+++ /dev/null
@@ -1,34 +0,0 @@
-{
-    "txid": "703b165ed20cffab0a5df5ad38b77a93b6e1e250e7a1d6d18162b546ab67f452",
-    "hash": "334256a59387797146757632b4c33bcfde0152b340e9a7639513cc25de6c32bc",
-    "version": 2,
-    "size": 190,
-    "locktime": 0,
-    "vin": [
-        {
-            "txid": "4d49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc59485",
-            "vout": 0,
-            "scriptSig": {
-                "asm": "3043021f302b36058b8617d81259d9da25e91b321eecf9d3826f36b83f7671572d0aa2022049b222b2e9d8356d22efcda07931d9360e2a4cdadabb62723463b5111d4a59ae[SINGLE|BIP341] 02c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5",
-                "hex": "463043021f302b36058b8617d81259d9da25e91b321eecf9d3826f36b83f7671572d0aa2022049b222b2e9d8356d22efcda07931d9360e2a4cdadabb62723463b5111d4a59ae632102c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5"
-            },
-            "sequence": 4294967295
-        }
-    ],
-    "vout": [
-        {
-            "value": 0.100000,
-            "n": 0,
-            "scriptPubKey": {
-                "asm": "OP_DUP OP_HASH160 06afd46bcdfd22ef94ac122aa11f241244a37ecc OP_EQUALVERIFY OP_CHECKSIG",
-                "hex": "76a91406afd46bcdfd22ef94ac122aa11f241244a37ecc88ac",
-                "reqSigs": 1,
-                "type": "pubkeyhash",
-                "addresses": [
-                    "1cMh228HTCiwS8ZsaakH8A8wze1JR5ZsP"
-                ]
-            }
-        }
-    ],
-    "hex": "02000000018594c5bdcaec8f06b78b596f31cd292a294fd031e24eec716f43dac91ea7494d0000000069463043021f302b36058b8617d81259d9da25e91b321eecf9d3826f36b83f7671572d0aa2022049b222b2e9d8356d22efcda07931d9360e2a4cdadabb62723463b5111d4a59ae632102c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5ffffffff01a0860100000000001976a91406afd46bcdfd22ef94ac122aa11f241244a37ecc88ac00000000"
-}
diff --git a/test/util/data/txcreatesignlotusv2-single-anyonecanpay.json b/test/util/data/txcreatesignlotusv2-single-anyonecanpay.json
index 0de437996..4baa7e97f 100644
--- a/test/util/data/txcreatesignlotusv2-single-anyonecanpay.json
+++ b/test/util/data/txcreatesignlotusv2-single-anyonecanpay.json
@@ -17,7 +17,7 @@
     ],
     "vout": [
         {
-            "value": 0.00100000,
+            "value": 0.100000,
             "n": 0,
             "scriptPubKey": {
                 "asm": "OP_DUP OP_HASH160 06afd46bcdfd22ef94ac122aa11f241244a37ecc OP_EQUALVERIFY OP_CHECKSIG",
diff --git a/test/util/data/txcreatesignlotusv2-single.json b/test/util/data/txcreatesignlotusv2-single.json
index b614ab033..837212468 100644
--- a/test/util/data/txcreatesignlotusv2-single.json
+++ b/test/util/data/txcreatesignlotusv2-single.json
@@ -17,7 +17,7 @@
     ],
     "vout": [
         {
-            "value": 0.00100000,
+            "value": 0.100000,
             "n": 0,
             "scriptPubKey": {
                 "asm": "OP_DUP OP_HASH160 06afd46bcdfd22ef94ac122aa11f241244a37ecc OP_EQUALVERIFY OP_CHECKSIG",
-- 
2.39.5 (Apple Git-154)

